### DISCUSSION TITLE
虚拟键盘

### DISCUSSION THREAD

**Discussion by jkiss** [ant-design/ant-design-mobile#4677] [Ideas]
既然都有虚拟数字键盘了，虚拟字母键盘不来一个？:stuck_out_tongue_winking_eye:

**Comment by awmleer** [comment]
可以考虑，不过优先级应该比较低，虚拟字母键盘实在是不太常见😅
而且，其实我们更推荐使用原生的键盘，除非是迫不得已

**Reply by jkiss** [reply]
哈哈，因为现在再写小程序，嵌的web-view，原生键盘关闭后，页面没有正常回到底部，底部有一块空白，上面又被切掉了，可能是小程序的性能问题

**Reply by awmleer** [reply]
这个看起来还是应该用别的方式来解决，做个虚拟键盘只能是绕过问题，解决不了问题

**Reply by jkiss** [reply]
嗯
